code/frameworks/nextjs/README.md

```diff
@@ -0,0 +1,622 @@
+# Storybook for Next.js <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+- [Supported Features](#supported-features)
+- [Requirements](#requirements)
+- [Getting Started](#getting-started)
+  - [In a project without Storybook](#in-a-project-without-storybook)
+  - [In a project with Storybook](#in-a-project-with-storybook)
+- [Documentation](#documentation)
+  - [Options](#options)
+  - [Next.js's Image Component](#nextjss-image-component)
+    - [Local Images](#local-images)
+    - [Remote Images](#remote-images)
+    - [Optimization](#optimization)
+    - [AVIF](#avif)
+  - [Next.js Routing](#nextjs-routing)
+    - [Overriding defaults](#overriding-defaults)
+    - [Global Defaults](#global-defaults)
+    - [Default Router](#default-router)
+    - [Actions Integration Caveats](#actions-integration-caveats)
+  - [Sass/Scss](#sassscss)
+  - [Css/Sass/Scss Modules](#csssassscss-modules)
+  - [Styled JSX](#styled-jsx)
+  - [Postcss](#postcss)
+  - [Absolute Imports](#absolute-imports)
+  - [Runtime Config](#runtime-config)
+  - [Custom Webpack Config](#custom-webpack-config)
+  - [Typescript](#typescript)
+  - [next.config.js](#nextconfigjs)
+    - [ESM](#esm)
+  - [Notes for Yarn v2 and v3 users](#notes-for-yarn-v2-and-v3-users)
+  - [FAQ](#faq)
+    - [Stories for pages](#stories-for-pages)
+    - [Statically imported images won't load](#statically-imported-images-wont-load)
+    - [This framework breaks when the .mjs extension for the next config is used](#this-framework-breaks-when-the-mjs-extension-for-the-next-config-is-used)
+    - [Module not found: Error: Can't resolve [package name]](#module-not-found-error-cant-resolve-package-name)
+- [Acknowledgements](#acknowledgements)
+
+## Supported Features
+
+👉 [Next.js's Image Component](#nextjss-image-component)
+
+👉 [Next.js Routing](#nextjs-routing)
+
+👉 [Sass/Scss](#sassscss)
+
+👉 [Css/Sass/Scss Modules](#csssassscss-modules)
+
+👉 [Styled JSX](#styled-jsx)
+
+👉 [Postcss](#postcss)
+
+👉 [Absolute Imports](#absolute-imports)
+
+👉 [Runtime Config](#runtime-config)
+
+👉 [Custom Webpack Config](#custom-webpack-config)
+
+👉 [Typescript](#typescript) (already supported out of the box by Storybook)
+
+## Requirements
+
+- [Next.js](https://nextjs.org/) >= 9.x
```

Comment:
Yes. Honestly, that's only because your addon already had support and it seemed straightforward to continue supporting.

I'm curious if other reviewers have opinions on the tradeoffs of broad support vs. less code to maintain, though...

Reply:
I can say that supporting v9 is definitely doable. It is probably a good idea to keep v9 support at least in the first major version of this so that people can more easily swap between my addon and this addon.

On the same line of thinking it probably is a good idea to make a point of supporting all the same features just to reduce friction from swapping 🤔

Reply:
We use the Next Image in our CLI template files. Since Next v9 does not support Next Images, we shouldn't try to support it and complicate our setup. @kylegach @yannbf